app/helpers/bookings_helper.rb

```diff
@@ -1,7 +1,74 @@
 module BookingsHelper
 
-  def fetch_booking_info
+  def update_saved(booking_params)
+    if @booking.update(booking_params)
+        @booking.save
+        message = "Update successful."
+    else
+      message = "Update failed."
+    end
+  end
+
+  def booking_params
+    params.require(:booking).
+    permit( :no_of_passengers,
+            :flight_id,
+            :confirmation_code,
+            :customer_id,
+            :id,
+            passengers_attributes:
+                                [:name,
+                                 :email,
+                                 :_destroy
+                                ] )
+  end
+
+  def flight_reseter(column, id, num)
+    booking = Booking.new
+    booking_info = booking.get_all_bookings(column, id)
+    {
+      code: booking_info[0]["flight_code"],
+      dest: booking_info[0]["destination"],
+      dept: booking_info[0]["departure"],
+      cost: booking_info[0]["cost"],
+      airline: booking_info[0]["airline"],
+      flight_datetime: booking_info[0]["dept_date"],
+      pass: num,
+      booking_id: booking_info[0]["id"]
+    }
+  end
+
+  def change_no_of_pass(passenger_num, id)
+    booking = Booking.new
+    num = params[:pass_num]
+    id = params[:id]
+    if booking.reset_passengers(num ,id)
+      flash[:notice] = "Change has been made, please enter passenger(s) information."
+      url_query = flight_reseter("b.id", id, num)
+      redirect_to book_path(url_query)
+    elsif !booking.reset_passengers(num ,id) && params[:id].nil?
+      flash[:notice] = "Could not update number of passengers, contact the admin."
+    end
+  end
+
+  def mail_worker(booking_params, name, email)
+    @confirmation_code = booking_params[:confirmation_code]
+    @passengers_list = booking_params[:passengers_attributes]
+    info = JSON.generate({
+        name: name,
+        email: email,
+        flight_id: booking_params[:flight_id]
+    })
```

Comment:
Indent the right brace the same as the first position after the preceding left parenthesis.